### DISCUSSION TITLE
HELP please locating EdgeTX 2.8.0 firmware for Radiomaster TX16S.

### DISCUSSION THREAD

**Discussion by UKFlyer-Dave** [EdgeTX/edgetx#5395] [Q&A] [answered]
I am having difficulty locating EdgeTX 2.8.0 firmware for Radiomaster TX16S.
I currently have OpenTX v2.3.15 on my TX, and understand that I can't migrate directly to EdgeTX v2.10.3, and have to load v2.8.0 first.
I have downloaded v2.8.0 SD card contents and sounds, but can't find the associated firmware zip file.
Any help and advice would be greatly appreciated.
Cheers, Dave

**Comment by pfeerick** [comment, marked as answer]
Any version of 2.8.x is fine, it doesn't need to specifically be 2.8.0.

https://github.com/EdgeTX/edgetx/releases/tag/v2.8.5

Or you can download the firmware, or flash via DFU directly via
https://buddy.edgetx.org/#/flash?version=v2.8.5&target=tx16s

**Reply by UKFlyer-Dave** [reply]
Thanks Peter for your reply.  It was a great help and I have now managed to fully upgrade my TX16S to EdgeTX 2.10 via the 2.8 release.
Cheers
Dave
